pkgs/applications/graphics/figma-linux/default.nix

```diff
@@ -17,7 +18,12 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-9UfyCqgsg9XAFyZ7V7TogkQou4x+ixFUfjXZ1/qlDmA=";
   };
 
-  nativeBuildInputs = [ autoPatchelfHook dpkg wrapGAppsHook3 ];
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
+    wrapGAppsHook3
```

Comment:
By adding `wrapGAppsHook3` and manually `wrapProgram`, you are wrapping `figma-linux` two times. See https://github.com/NixOS/nixpkgs/blob/nixos-unstable/doc/languages-frameworks/gnome.section.md#when-using-wrapgapps-hook-with-special-derivers-you-can-end-up-with-double-wrapped-binaries-ssec-gnome-common-issues-double-wrapped